README.md

```diff
@@ -1,7 +1,10 @@
 # preact-render-to-string
 
 [![NPM](http://img.shields.io/npm/v/preact-render-to-string.svg)](https://www.npmjs.com/package/preact-render-to-string)
-[![travis-ci](https://travis-ci.org/preactjs/preact-render-to-string.svg)](https://travis-ci.org/preactjs/preact-render-to-string)
+[![Build status][github-ci-badge]][github-ci-url]
+
+[github-ci-badge]: https://github.com/preactjs/preact-render-to-string/actions/workflows/ci.yml/badge.svg
+[github-ci-url]: https://github.com/preactjs/preact-render-to-string/actions/workflows/ci.yml
```

Comment:
```suggestion
[![Build status](https://github.com/preactjs/preact-render-to-string/actions/workflows/ci.yml/badge.svg)](https://github.com/preactjs/preact-render-to-string/actions/workflows/ci.yml)
```

Good catch, thanks!

I don't think there's any real need to extract out the links, and it creates inconsistency (the NPM badge right above is still all inline)

Reply:
Updated as suggested.